README.md

```diff
@@ -16,11 +16,12 @@ The following configuration runs a DHCPv6-only server, listening on all the inte
 
 ```
 server6:
-    listen: '[::]:547'
+    listen: '[::]:547'       # this listens on every interface
+    # - "[fe80::1%eth0]:547" # this will listen on eth0
     plugins:
-        - server_id: LL 00:de:ad:be:ef:00
-        - file: "leases.txt"
-        # - dns: 8.8.8.8 8.8.4.4 2001:4860:4860::8888 2001:4860:4860::8844
+        - server_id: LL 00:de:ad:be:ef:00  # set the server ID (option 2)
+        - file: "leases.txt"  # read static lease configuration from leases.txt
+        # - dns: 8.8.8.8 8.8.4.4 2001:4860:4860::8888 2001:4860:4860::8844  # set DNS (option 23)
 
 #server4:
 #    listen: '0.0.0.0:67'
```

Comment:
Probably worth adding a v4 equivalent as well, eg `192.168.1.1%eth0`, or even `%eth0`

Reply:
I thought about it - but the v4 example is commented out. I'm thinking that this is not the right thing to do, and we should put in the readme the simplest example. The example config is already linked immediately after, and we should be probably clear that it's where people should look for for more information

Reply:
in other words I'm for reverting the README change and reinforce the wording to look at the example for more details

Reply:
oh, yes, that would be simpler to manage and remember to update this way